fastlane/Fastfile

```diff
@@ -714,11 +714,13 @@ ENV["validate_translations"]="lintVanillaRelease"
     version = options[:version]
     apk_file_path = universal_apk_file_path(version)
     aab_file_path = bundle_file_path(version)
+    # APKs built on CI are unsigned, so don't upload them until we have a fix
+    release_assets = is_ci ? aab_file_path : "#{apk_file_path},#{aab_file_path}" 
```

Comment:
I'm unsure if it makes sense to have a different behavior depending on if we create the release from CI vs a local machine tho, as that might mean that some releases (created locally) would have the (signed) apk but some (created from CI) would not, which… might create as much confusion as having the unsigned apk.

What do you think about always just uploading the aab for consistency?

Reply:
If we create the release on a local machine, the APK is signed, so it can be downloaded and installed without any additional step. I think that if we have a valid APK, it's worth it to upload it. 
I don't think it would add much confusion: we may still continue to manually upload some APKs for some testing purposes anyway, so we'll likely have some releases with the APK, and others without. 
While this is not perfect, I think that as long as the APK you download is installable, there shouldn't be big issues.